openslides/core/static/js/core/projector.js

```diff
@@ -150,7 +150,8 @@ angular.module('OpenSlidesApp.core.projector', ['OpenSlidesApp.core'])
     'slides',
     'Config',
     'ProjectorID',
-    function($scope, $location, $timeout, Projector, slides, Config, ProjectorID) {
+    'Logos',
```

Comment:
I am sorry but now I see that this is a problem. We make mediafile a dependency of core. Please let us think about a way not to add such a dep path.